inst/app/ui.R

```diff
@@ -37,13 +37,14 @@ tagList(
           tags$br()
         ),
         mainPanel(
-          DiagrammeR::grVizOutput("flowchart", width = "600px", height = "600px"),
+          DiagrammeR::grVizOutput("flowchart", width = "800px", height = "800px"),
           textOutput("clickMessage")
         ),
       )
     )
   ),
   tags$head(
+    tags$link(rel = "icon", type = "image/x-icon", href = "pandora-isomemo.ico"),
```

Comment:
I added an icon for the browser tab. This removed an error when inspecting the app after loading.